core/execution/future/market.go

```diff
@@ -1654,6 +1655,7 @@ func (m *Market) leaveAuction(ctx context.Context, now time.Time) {
 
 	if wasOpeningAuction && !m.as.IsOpeningAuction() && m.getCurrentMarkPrice().IsZero() {
 		m.markPriceCalculator.OverridePrice(m.lastTradedPrice)
+		m.pMonitor.ResetPriceHistory([]*types.Trade{{Price: m.lastTradedPrice, Size: 1}})
```

Comment:
Does the uncross volume matter in any way here? We would be discarding the external price data entirely, so the next time around, we could end up calculating a mark price that is perhaps more likely to breach a monitoring bound. Not really stating that as a fact, I just wonder if this reset call would essentially prime the price monitoring bounds as though the only price data available is the last traded price, but next time around the mark price calculator could end up providing a significantly different price

Reply:
The volume only matters within the same timestamp, but there all trades will be at the same price so doesn't matter. 

The reset only happens when mark price was 0 after opening auction, so better to have last traded price than 0.